test/component/uk/gov/hmrc/individualsemploymentsapi/controller/v2/LiveEmploymentsControllerSpec.scala

```diff
@@ -203,11 +203,14 @@ class LiveEmploymentsControllerSpec extends BaseSpec {
     }
 
     scenario("invalid match id") {
+
+      val employerRef = "247%2FZT6767895A"
```

Comment:
Do we expect the consumer to url encode the amp ref? If so have we stated this in the docs anywhere? I have asked a question of Wayne as his example in the schema does not include the `/`. We may end up having to strip this out if its not required.